tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -96,6 +97,7 @@ private TiSession(TiConfiguration conf) {
     }
     this.regionManager = null;
     this.clientBuilder = null;
+    refreshNewCollationEnabled();
```

Comment:
refreshNewCollationEnabled needs tiSession to get the version. And it could be put in the ClientSession constructor function, which is existed in the GSOC.

Reply:
refreshNewCollationEnabled needs tiSession to get the version. And it could be put in the ClientSession constructor function, which is existed in the GSOC.